src/couch_replicator_sup.erl

```diff
@@ -13,43 +13,60 @@
 
 -module(couch_replicator_sup).
 -behaviour(supervisor).
--export([start_link/0, init/1]).
+-export([start_link/0, init/1, restart_mdb_listener/0]).
 
 start_link() ->
     supervisor:start_link({local, ?MODULE}, ?MODULE, []).
 
 init(_Args) ->
+    MdbChangesArgs = [
+        <<"_replicator">>,  % DbSuffix
+        couch_replicator,   % Module
+        nil,                % Callback context
+        [skip_ddocs]        % Options
+    ],
     Children = [
-        {couch_replicator_scheduler,
-            {couch_replicator_scheduler, start_link, []},
-            permanent,
-            brutal_kill,
-            worker,
-            [couch_replicator_scheduler]},
         {couch_replicator_scheduler_sup,
             {couch_replicator_scheduler_sup, start_link, []},
             permanent,
             infinity,
             supervisor,
             [couch_replicator_scheduler_sup]},
+        {couch_replicator_scheduler,
```

Comment:
Because it is a rest_for_one supervisor. The scheduler_sup should be up before scheduler itself so it can receive calls from replicator scheduler. For example it calls ` couch_replicator_scheduler_sup:start_child()` so _sup should be up by then.